lib/modules/datasource/custom/index.spec.ts

```diff
@@ -198,6 +198,41 @@ describe('modules/datasource/custom/index', () => {
       expect(result).toBeNull();
     });
 
+    it('return releases for yaml API directly exposing in Renovate format', async () => {
+      const expected = {
+        releases: [
+          {
+            version: '1.0.0',
+          },
+          {
+            version: '2.0.0',
+          },
+          {
+            version: '3.0.0',
+          },
+        ],
+      };
+
+      const yaml = `releases:\n  - version: 1.0.0\n  - version: 2.0.0\n  - version: 3.0.0`;
```

Comment:
Use a codeblock to make this more readable https://github.com/renovatebot/renovate/blob/2ce4fc5b06055a08108fbb5c5437f522d8371097/lib/util/exec/hermit.spec.ts#L65